apps/web/core/components/features/projects/view-project-info-modal.tsx

```
@@ -160,14 +161,15 @@ export function ViewProjectInfoModal(props: Readonly<IViewProjectInfoModalProps>
-						<Section title={t('common.DESCRIPTION')}>
-							<div className="p-3 text-sm text-gray-700 whitespace-pre-wrap rounded-lg bg-gray-50 dark:bg-dark--theme dark:text-gray-300">
-								{project.description}
-							</div>
-						</Section>
-					)}
+				{/* ===== DESCRIPTION ===== */}
+				{project.description && (
+					<Section title={t('common.DESCRIPTION')}>
+						<div
+							className="p-3 text-sm text-gray-700 whitespace-pre-wrap rounded-lg bg-gray-50 dark:bg-dark--theme dark:text-gray-300 [&_strong]:font-bold [&_em]:italic [&_u]:underline [&_code]:bg-gray-200 [&_code]:dark:bg-gray-700 [&_code]:px-1 [&_code]:rounded [&_p]:mb-1 [&_p:last-child]:mb-0"
+							dangerouslySetInnerHTML={{ __html: sanitizeHtml(project.description) }}
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)